docs/conf.py

```diff
@@ -26,11 +26,16 @@
 
 # Add any Sphinx extension module names here, as strings. They can be extensions
 # coming with Sphinx (named 'sphinx.ext.*') or your custom ones.
-extensions = ['sphinx.ext.autodoc', 'sphinx.ext.intersphinx',
-              #'sphinx.ext.autosummary', 'sphinx.ext.pngmath',
+extensions = ['sphinx.ext.autodoc',
+              #'sphinx.ext.autosummary',
               'numpydoc',
               'imageio_ext', ]
 
+# Monkey-patch numpydoc to don't do the autosummary thing
+import docscrape_sphinx
+assert docscrape_sphinx.SphinxDocString._str_member_list
+docscrape_sphinx.SphinxDocString._str_member_list = lambda self, name: []
```

Comment:
@ghisvail this disables the autosummary thing in a way that should also work with a system numpydoc. We're monkey-patching, which isn't great, especially since its a private method, but this seems the only way.